### PR TITLE
Add presubmit jobs for node-feature-discovery

### PR DESCRIPTION
**Pull request by marquiz**
Add rudimentary jobs for CI testing.

### CODE REVIEW

File: config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - name: pull-node-feature-discovery-verify
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-discovery
+      testgrid-tab-name: verify
+      description: "Verify source code of node-feature-discovery: check formatting, lint etc."
+    spec:
+      containers:
+      - image: golang:1.13
```
Comment (ArangoGutierrez) [inline config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits.yaml:12]: now that we have a PR for bumping to 1.14, we should also bump it here

File: config/jobs/kubernetes-sigs/node-feature-discovery/OWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- kad
+- marquiz
```
Comment (spiffxp) [inline config/jobs/kubernetes-sigs/node-feature-discovery/OWNERS:5]: So they will get assigned for review by blunderbuss
```suggestion
approvers:
- kad
- marquiz
reviewers:
- kad
- marquiz
```

Comment (marquiz) [reply config/jobs/kubernetes-sigs/node-feature-discovery/OWNERS:5]: OK, fixed

File: config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits.yaml (hunk 1 above)
Comment (marquiz) [reply config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits.yaml]: Good point! Now updated to 1.14

**Comment by marquiz** [comment]
I pushed a new version of the PR with the comments addressed

**Pull request merged**